docs/jwst/references_general/apcorr_reffile.inc

```diff
@@ -263,18 +240,55 @@ The row selection criteria for each instrument+mode are:
 * MIRI:
    - Image: Filter and Subarray
    - LRS: Subarray
-   - MRS: None (table contains a single row)
 * NIRCam:
    - Image: Filter and Pupil
    - WFSS: Filter and Pupil
 * NIRISS:
    - Image: Filter and Pupil
    - WFSS: Filter and Pupil
 * NIRSpec:
-   - IFU and MOS: Filter and Grating
+   - MOS: Filter and Grating
    - Fixed Slits: Filter, Grating, and Slit name
 
 Note: Spectroscopic mode reference files contain the "nelem_wl" and "nelem_size"
 entries, which indicate to the pipeline step how many valid elements are contained
 in the "wavelength" and "size" arrays, respectively. Only the first "nelem_wl"
 and "nelem_size" entries are read from each array.
+
+
+IFU APCORR Reference File ASDF Format
++++++++++++++++++++++++++++++++++++++
+For IFU data the APCORR reference files are in ASDF format. The aperture correction
+varies with wavelength and the contents of the files are shown below.
+
+:Data model: `~jwst.datamodels.MirMrsApcorrModel`
+
++------------+-------+-------------+-----------+------------+----------+
+| Instrument | Mode  | Column name | Data type | Dimensions | Units    |
++============+=======+=============+===========+============+==========+
+| MIRI       | MRS   | wavelength  | float     | 1D array   | micron   |
++            +       +-------------+-----------+------------+----------+
+|            |       | radius      | float     | 1D array   | arcsec   |
```

Comment:
radius, apcor, and apcorr_err are all 2d arrays.  At the moment the 2nd dimension doesn't add information, but in future it could be used to provide different aperture corrections for different radii.